stubs/inertia/resources/js/Jetstream/Dropdown.vue

```diff
@@ -55,10 +55,6 @@
                 }
             }
 
-            this.$once('hook:destroyed', () => {
-                document.removeEventListener('keydown', closeOnEscape)
-            })
-
```

Comment:
So where does this get removed now?

Reply:
Whoops totally forgot about that, I will add it tommorow, this hook removed at first because the depreciation of `$once` function
but this willbe moved in the `beforeUnmount` lifecycle hook without the `$once` function